packages/dappmanager/src/modules/compose/unsafeCompose.ts

```diff
@@ -64,9 +65,11 @@ export function parseUnsafeCompose(
   const dnpName = manifest.name;
   const version = manifest.version;
   const isCore = getIsCore(manifest);
+  // Use of new compose feature "name"only available in version 3.5
+  // https://docs.docker.com/compose/compose-file/compose-file-v3/#name-1
 
   return cleanCompose({
-    version: composeUnsafe.version || "3.4",
+    version: sanitizeComposeFileVersion(composeUnsafe.version),
```

Comment:
What does it mean to sanitize a version? The function ensures that version is at least a minimum. Call it that